### ISSUE TITLE
Disable rules via config file

### ISSUE THREAD

**Issue by xoxys** [DavidAnson/markdownlint#802]
Looks like disabling rules via config file doesn't seem to work anymore. The following `.markdownlint.yml` config file stopped working:

```YAML
---
default: True
MD013: False
MD041: False
MD042: False
MD024: False
```

> exampleSite/content/posts/advanced/shortcodes.md:41:81 MD013/line-length Line length [Expected: 80; Actual: 105]

I'm using markdownlint via `igorshubovych/markdownlint-cli`

**Comment by xoxys** [comment]
The vscode extension is affected as well.

**Comment by xoxys** [comment]
As alway, after creating the issue I found the problem... I had a `<!-- markdownlint-enable -->`  instead of `<!-- markdownlint-restore -->` in a file... Sorry!
